dist_tables/ddl.rst

```diff
@@ -61,6 +61,45 @@ Each created shard is assigned a unique shard id and all its replicas have the s
 
 You are now ready to insert data into the distributed table and run queries on it. You can also learn more about the UDF used in this section in the :ref:`user_defined_functions` of our documentation.
 
+.. _colocation_groups:
+
+Co-Location Groups
+------------------
+
+By default two tables are co-located when they are distributed by columns of the same type using the same distribution method and over the same number of shards. Another way to say this is that the tables are in the same *co-location group.*
+
+It is important for performance to have full control over co-location groups. All members of a co-location group support router-executed JOIN queries between one another on their distribution column. They also support foreign key constraints for keys containing the distribution column. Additionally shard rebalancing preserves co-location in new shard placements.
```

Comment:
"All members of a co-location group support router-executed JOIN queries between one another on their distribution column."

This sentence talks about how colocation groups provide benefits. Could we instead mention the benefit from the user's perspective? This is hard because colocation provides slightly different benefits depending on the use case. If you have a multi-tenant application (router executor), you get the following benefits:

* Transactions scoped to a tenant
* Joins across all tables for a tenant
* Foreign key constraints to ensure data integrity, etc.

If we wanted to be more generic, we could say that when tables are colocated and the user runs a join operation, the join executes faster because it doesn't need to shuffle data over the network.